### ISSUE TITLE
Windows binaries for v3.0.12 - will they be made available?

### ISSUE THREAD

**Issue by erikvullings** [mapnik/mapnik#3523]
On the [release page](http://mapnik.org/pages/downloads.html), you mention that 'We plan on providing prebuilt binaries soon for Mapnik 3.0.12.' That was written about 3 weeks ago. So will you make a Windows binary version available too, and if so, when can we expect it (as I'm not really looking forward to build it from scratch)?

**Comment by donnyv** [comment]
Yeah I just noticed that too. The last binary build was in 2013?? 3yrs ago?

**Comment by donnyv** [comment]
So I found this thread about binaries and how this poor dude went out on a limb to use Mapnik for windows and couldn't get it compiled. Someone posted a link to get a compiled one and the link
sends you to a yml file that was changed to remove the value that was needed......strange.

The value is just the release version #.

So the newest one is located here.
https://mapnik.s3.amazonaws.com/dist/dev/mapnik-win-sdk-v3.0.12-x64-14.0.7z

Not sure why this isn't included in the release.
